examples/albef/data/transforms.py

```diff
@@ -9,16 +9,9 @@
 
 import torch
 
-from torchtext.transforms import (
-    AddToken,
-    BERTTokenizer,
-    PadTransform,
-    Sequential,
-    StrToIntTransform,
-    ToTensor,
-    Truncate,
-)
+from torchtext.transforms import PadTransform, Sequential, ToTensor, Truncate
 from torchvision import transforms
+from transformers.models.bert.tokenization_bert import BertTokenizer
```

Comment:
do we know the root cause for torchtext one not working?

Reply:
Idk the root cause 100%, but it is related to the absence of the UNK token, which I cannot access from TorchText's BERTTokenizer API. It is weird cause I see it training on multiple GPUs but could not reproduce the issue when running locally